daisy_workflows/image_import/debian/translate.py

```diff
@@ -51,17 +52,17 @@ def DistroSpecific(g):
     utils.update_apt(g)
     utils.install_apt_packages(g, 'gnupg')
 
-    g.command(
+    run(g,
```

Comment:
Which run invocations should have check=True and which should have it False? It's not clear how this choice is made now or for future run invocations we might add.

Reply:
`check=True` is good for commands where you don't inspect the output and you want the failure to propagate automatically. For example: `yum install package`; if it's successful, you don't need the output from yum.

Reply:
This would be useful if documented in the function description so developers working on this in the future know how to use it.

Reply:
sgtm - done!